src/automations/Job.ts

```diff
@@ -311,6 +316,28 @@ export default class Job {
         return jobs;
     }
 
+    public async hasAccess(jobID: number) {
+        const url = joinURL(MAIN_URL, "/alljobs");
+        await this.page.goto(url);
+
+        await this.loadAllJobs();
+        const jobElements = await this.page.$$(".job");
+        if (!jobElements || !jobElements.length)
+            throw new Error("No job found.");
+
+        for (const jobElement of jobElements) {
+            const nameCell = await jobElement.$(".job-name");
+            if (!nameCell) throw new Error("Cannot get job name cell.");
+
+            const jobIDFromCell = await getIDFromURL(nameCell, "a");
+            if (jobIDFromCell === jobID) {
+                const isRecruiter = await this.isRecruiter(jobElement);
```

Comment:
I think you can delete the extra assignment:
```suggestion
                return await this.isRecruiter(jobElement);
```